README.md

```diff
@@ -1,2 +1,208 @@
-# phplist-plugin-simplesaml
+# phpList simpleSAML Plugin
+
 SimpleSaml plugin for phpList
+
+## Plugin Setup
+
+After `cd`-ing into the configured phpList plugin directory:
+
+- `git clone https://github.com/phpList/phplist-plugin-simplesaml.git`
+- `composer install`
+- `cp .env.example. .env`
+- Ensure `SIMPLESAMLPHP_INSTALLTATION_PATH` in `.env` is the correct relative path installation of `simplesamlphp`
+
+# SimpleSAMLPHP Setup
+
+- `cd phplist-plugin-simplesaml/plugins/authsaml/simplesamlphp`
+- `composer install`
+- `npm install`
+- `npm run build`
+
+NB : `simplesamlphp` **MUST BE CONFIGURED ON SAME DOMAIN** as your phplist installation. Hence the right place to do the clone is in `PATH_TO_PHPLIST_INSTALLATION/public_html`
+
+You might be required to manually create a `log` directory with write permissions granted, so: `mkdir log && sudo chmod -R a+rwx log`
+
+## Configuration
+
+In `PATH_TO_PHPLIST_INSTALLATION/public_html/simplesamlphp` the following directories should be present.
```

Comment:
This can be done in the plugin, eg plugins/authsaml/simplesamlphp/

Reply:
Instead of bundling `simplesamlphp` code, do mind me adding this as a submodule, that user can just pull from git?

Reply:
Done.